### ISSUE TITLE
Provide feedback about SeRA API to SeRA team

### ISSUE THREAD

**Issue by mjgiarlo** [sul-dlss/rialto#145]
The early [SeRA API documentation](https://asconfluence.stanford.edu/confluence/display/MaIS/SeRA+API+-+End+User+Documentation) has been shared with us now.

So far, the API supports a single GET method, to retrieve all SeRA (grant) records for a given SUNetID
that means it will be on us to loop over all our SUNetIDs and pull back related grant information. we may want to provide them feedback about this. (Have they considered a batch interface, pagination, etc.?)

Note that the SeRA data comes in three different scopes or access levels. If our webapp will be publicly visible, we will only have access to the limited information available in the `sera.public` scope. To get `sera.stanford-only` data (dept name, school name, sponsor name), our servers and apps will need to meet ISO "Moderate Risk" requirements. To get information about the award amounts, SeRA requires that you only provide that to the PI and no one else (this is the `sera.private` scope).

Long story short, I'm wondering how useful this API, as currently described, is going to be for RIALTO.

**Comment by peetucket** [comment]
* currently in order to pull all of the data for everyone at Stanford into RIALTO, we need to iterate and make one call per SUNETID - having the ability to request more data per call would be more efficient
* on a related note, it would be very helpful and more efficient to "filter by date updated" parameter so that if we do routine pulls of data, we can only pull new data and not reprocess everything
* is it possible or useful to make requests by employee id instead of sunetid?   is their data linked to people that don't have a sunet?
* when iterating through all our records, we got a single 500 response for the user with sunet id "jackler" - not sure if maybe there is a data error in there or if it was an intermittent server issue
* can you remind us what the edwUpdateDateTime (e.g. "24-FEB-14") is, is this the date when the record was last changed?
* field formatting: while consumers can parse, it would be nice to standardize (e.g. dates/times as ISO 8601 dates, booleans instead of "Y/N" for flag fields, etc.)
